index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div>
+            <a href="#" class="logo">My Logo</a>
+        </div>
+        <nav class="menu">
+            <img src="./assets/icons/Union.png" alt="Menu Icon">
+        </nav>
+    </header>
+
+    <main>
+        <section class="headline-section">
+            <h1 class="heading-1">I'm Abdul Glad to see you!</h1>
+            <p class="intro-info">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <h2 class="connect-info">LET’S CONNECT</h2>
+
+            <ul class="main-icons">
+                <li><a href="twitter.com"><img class="logo-images" src="./assets/icons/Twitter_logo.svg" alt="Twitter_logo"></a></li>
+                <li><a href="linkedin.com"><img class="logo-images" src="./assets/icons/Linkedin_icon.svg" alt="Linkedin Icon"></a></li>
+                <li><a href="medium.com"><img class="logo-images" src="./assets/icons/Medium.svg" alt="Medium logo"></a></li>
+                <li><a href="Github.com"><img class="logo-images" src="./assets/icons/Github_icon.svg" alt="Github logo"></a></li>
+                <li><a href="angelist.com"><img class="logo-images" src="./assets/icons/Angelist.svg" alt="Angelist logo"></a></li>
+            </ul>
```

Comment:
- [ ] please try to add the exact  padding or margin between icons in order to match the Design right now yours are a bit closer
- Yours 
![image](https://user-images.githubusercontent.com/61790587/170540199-b2283b35-1c6b-42a0-85d9-c43f85baacee.png)
- Figma 
![image](https://user-images.githubusercontent.com/61790587/170540226-98d0eb7f-6691-4c76-af11-8f90c2ff7f0c.png)